spinn_front_end_common/utilities/__init__.py

```diff
@@ -15,5 +15,6 @@
 
 from .failed_state import FailedState
 from .simulator_interface import SimulatorInterface
+from .iobuf_extractor import IOBufExtractor
 
-__all__ = ["FailedState", "SimulatorInterface"]
+__all__ = ["FailedState", "SimulatorInterface", "IOBufExtractor"]
```

Comment:
wheres emergency recovery?

Reply:
Don't know - wondering what the importer calls...